ctgan/synthesizer.py

```diff
@@ -0,0 +1,209 @@
+import numpy as np
+import torch
+from torch import optim
+from torch.nn import functional
+
+from ctgan.conditional import ConditionalGenerator
+from ctgan.models import Discriminator, Generator
+from ctgan.sampler import Sampler
+from ctgan.transformer import DataTransformer
+
+
+class CTGANSynthesizer(object):
+    """docstring for IdentitySynthesizer."""
```

Comment:
Add docstring for this class. Clarify the input and output type. For example, both pandas.DataFrame and np.array are supported.

Reply:
I added docstrings to the class and its public methods, `fit` and `sample`.
More docstrings can be added in a later PR, when the rest of the modules are reviewed.